web_external/js/utilities.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Miscellaneous utility functions.
+ */
+
+/**
+ * Show an alert dialog with a single button.
+ * @param [text] The text to display.
+ * @param [buttonText] The text for the button.
+ * @param [buttonClass] Class string to apply to the button.
+ * @param [escapedHtml] If you want to render the text as HTML rather than
+ *        plain text, set this to true to acknowledge that you have escaped any
+ *        user-created data within the text to prevent XSS exploits.
+ * @param callback Callback function called when the user clicks the button.
+ */
+isic.showAlertDialog = function (params) {
```

Comment:
This is modeled off [girder.confirm]( https://github.com/girder/girder/blob/599a9dbe7180f3f02bb7605205ec1c88f92fd368/clients/web/src/utilities/MiscFunctions.js#L108).